lab-doug/lib/storage.js

```diff
@@ -0,0 +1,40 @@
+'use strict';
```

Comment:
For lab 8, we wanted you to bring in del and mkdirp as dependencies to make all the directory creation and deletion easier.